charts/opentelemetry-collector/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-collector
-version: 0.61.2
+version: 0.62.0
```

Comment:
@jorgeancal this should be 0.62.1 since we just merged the latest collector triggering a minor version bump and your pr is not smth breaking but just a bug fix.